drf_to_mkdoc/utils/schema.py

```
@@ -5,81 +5,200 @@
+                    attrs[field_name] = {
+                        "type": "list",
+                        "child_serializer": f"{child_class.__module__}.{child_class.__name__}",
+                    }
+                elif hasattr(field, "serializer"):
+                    # Handle nested serializers
+                    nested_class = field.serializer.__class__
+                    attrs[field_name] = {
+                        "type": "nested",
+                        "serializer": f"{nested_class.__module__}.{nested_class.__name__}",
+                    }
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)